components/library/babl/babl.p5m

```diff
@@ -59,30 +59,30 @@ file path=usr/lib/$(MACH64)/babl-0.1/sse4-int8.so
 file path=usr/lib/$(MACH64)/babl-0.1/two-table.so
 file path=usr/lib/$(MACH64)/babl-0.1/u16.so
 file path=usr/lib/$(MACH64)/babl-0.1/u32.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-CIE.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-cairo.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-double.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-float.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-gegl-fixups.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-gggl-lies.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-gggl.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-grey.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-simple.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-u16.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-u32.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-ycbcr.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-CIE.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-cairo.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-double.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-float.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-gegl-fixups.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-gggl-lies.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-gggl.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-grey.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-simple.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-u16.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-u32.so
-file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v3-ycbcr.so
+file path=usr/lib/$(MACH64)/babl-0.1/x86-64-v2-CIE.so variant.arch=i386
```

Comment:
Please do this using a transform to keep changes between this and the sample manifest as minimal as possible.

Thank you.